src/utils/fetch.js

```diff
@@ -237,6 +237,17 @@ function localFetch(method, url, options) {
         );
 }
 
+// Check for handled type: https://fetch.spec.whatwg.org/#bodyinit
+// URLSearchParams and ReadableStream are guarded, because they are not supported in IE
+// USVString is not handled because it will be typeof "string"
+function isAlreadySupported(body) {

```

Comment:
Please add unit tests covering scenarios where supported body type is provided and it's not getting stringified.

Reply:
Unit tests added.
I also had to update `package-lock.json` and change `testEnvironment` to `jsdom`.